.github/workflows/merge-to-main.yml

```diff
@@ -12,6 +12,12 @@ concurrency:
 jobs:
   create-pr:
     name: Update PR or Release
+    permissions:
+      id-token: write
+      contents: write
+      packages: write
```

Comment:
`packages: write` shouldn't be needed as that's for the GitHub package registry which we migrated away from.

Reply:
```suggestion
```